packages/jasmine/src/jasmine_node_test.bzl

```diff
@@ -51,8 +51,22 @@ def jasmine_node_test(
         tags = tags,
     )
 
-    all_data = data + srcs + deps + [jasmine]
-    all_data += [Label("//:src/jasmine_runner.js")]
+    runner_name = name + "." + "runner"
+
+    nodejs_binary(
```

Comment:
Any particular reason to wrap this in a `nodejs_binary` compared to how it was before?